syft/grid/autoscale/test.py

```diff
@@ -6,8 +6,12 @@
     credentials="/usr/terraform.json", project_id="project", region="us-central1",
 )
 
-NEW.compute_instance(
-    name="new-12345", machine_type="f1-micro", zone="us-central1-a", image_family="debian-9",
```

Comment:
Would it be possible to pass these in as Enums we provide instead of strings? That way we could get better autocomplete for it you know? (as opposed to having to know the *exact* string of what you need)

Reply:
(Very nice PR btw! Thank you for contributing!)

Reply:
@iamtrask Added Enums, please take a look, no need of looking up the exact string now.
Just added 1 value in each category, will add more later.

Reply:
@hericlesme So basically I need to get rid of the Enum module and capitalize everything?

Reply:
@hericlesme @iamtrask I got rid of the enum module and added more enums, please take a look.